src/components/nav/CustomNavbar.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Navbar as RSNavbar} from 'reactstrap';
+
+const CustomNavbar = props => {
+  const {children, ...otherProps} = props;
+
+  return <RSNavbar {...otherProps}>{children}</RSNavbar>;
+};
+
+CustomNavbar.defaultProps = {
+  fluid: false,
+  color: 'light',
```

Comment:
There's a bit of an issue here (and also with `Navbar`) I think with how the background colour is set. The `color` prop is used by reactstrap to add `bg-${color}` to the classes. These classes use the `!important` rule to set the background colour which means to set the background colour to something custom using either CSS classes or inline style arguments you need to pass something like `color=False` to prevent `bg-light` being applied by default.

I think there are two options:
1. Remove the default argument, forcing people to explicitly state a colour rather than relying on defaults.
2. Allow a hex code to be passed to `color`. We would check the `color` prop before passing it on to the underlying reactstrap component or using it to set the `background-color` style argument as appropriate.

Of course these are not mutually exclusive and could also be combined.

@pbugnion what do you think?

Reply:
I've implemented option 2. for now, it works quite well I think, and in case people really want to do something very custom they always have the option of passing `color=False` to switch everything off entirely.

Reply:
I think option 2 is the right call.